pkg/manager/internal.go

```diff
@@ -485,8 +485,8 @@ func (cm *controllerManager) engageStopProcedure(stopComplete <-chan struct{}) e
 				cm.internalCancel()
 			})
 			select {
-			case err, ok := <-cm.errChan:
-				if ok {
+			case err := <-cm.errChan:
```

Comment:
Wondering if this actually fixes the issues.

Isn't it a bit random if the err is read here or in l.448?

Reply:
But maybe good enough, whoever is calling manager.Start can also decide ignoring Context.Canceled

Reply:
Just realized that engageStopProcedure is only called after start returns, so all good